homework_03/14_martin_damyanov/homework.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+import math
+
+class RelativePosition(Enum):
+    NO_COMMON_POINTS = 1
+    TOUCHING = 2
+    INTERSECTING = 3
+    SAME = 4
+
+def distance(a, b):
+    try:
+        assert isinstance(a, Point)
+        assert isinstance(b, Point)
+    except:
+        print("Given arguments aren't points")
+        exit()
```

Comment:
I like the idea of the try-catch block, printing an error message, so I do not have any obligations for the assertions here!